packages/paste-core/components/tabs/src/TabList.tsx

```diff
@@ -1,13 +1,58 @@
 import { Box } from "@twilio-paste/box";
 import type { BoxProps } from "@twilio-paste/box";
+import { css, styled } from "@twilio-paste/styling-library";
 import { TabPrimitiveList } from "@twilio-paste/tabs-primitive";
+import type { ThemeShape } from "@twilio-paste/theme";
 import type { HTMLPasteProps } from "@twilio-paste/types";
 import * as React from "react";
 
 import { TabsContext } from "./TabsContext";
 import type { Variants } from "./types";
 import { getElementName } from "./utils";
 
+/**
+ * This wrapper applies styles that customize the scrollbar and its track.
+ */
+const SidebarNavigationWrapper = styled.div(({ theme }: { theme: ThemeShape }) => {
+  const { colorBackgroundStronger, colorBackgroundInverseStronger } = theme.backgroundColors;
+
+  return css({
+    paddingBottom: "4px",
```

Comment:
issue: This isn't adding 4px extra space below the scrollbar:

<img width="1149" alt="Screenshot 2024-04-11 at 2 48 54 PM" src="https://github.com/twilio-labs/paste/assets/3821840/de5dcdd5-1bfb-4963-b2ca-157c649b7ca2">

(Screenshot in Chrome)

Reply:
The padding is for the gap between the tab list and the scrollbar.

The gap between the tab list and the tab content is built into the TabPanel component.

Reply:
welp, wherever it needs to be, there should be an extra 4px between the scrollbar and the TabPanel so they're not right against each other when TabPanel paddingTop is 0

Reply:
Oh I see, ok